src/amr/data/particles/particles_data.h

```diff
@@ -417,96 +368,145 @@ namespace amr
         //! end index"
         SAMRAI::hier::Box interiorLocalBox_;
 
+        void copy_(SAMRAI::hier::Box const& overlapBox, ParticlesData const& sourceData)
+        {
+            auto myDomainBox         = this->getBox();
+            auto& srcDomainParticles = sourceData.domainParticles;
 
+            PHARE_LOG_START("ParticleData::copy_ DomainToDomain");
+
+            // first copy particles that fall into our domain array
+            // they can come from the source domain or patch ghost
+            auto destBox  = myDomainBox * overlapBox;
+            auto new_size = domainParticles.size();
+
+            if (!destBox.empty())
+            {
+                auto destBox_p = toPHAREBox<dim>(destBox);
```

Comment:
toPHAREBox was removed in #629

Reply:
https://gist.githubusercontent.com/PhilipDeegan/017af0ae9fe254a8967eb783243998a3/raw/ac908b7cc3ac871017972220a78182edf7a1e145/gistfile1.txt :eyes: